src/app/store/chats-store/chats.actions.ts

```diff
@@ -0,0 +1,24 @@
+import { createAction, props } from '@ngrx/store';
+import MessageInterface from '../../interfaces/chats';
+
+export const getOldMessages = createAction('[Chat Component] Get all messages');
```

Comment:
You can just prefix the tags with "[Chat]" because naming it "[Chat Component]" either makes it only limited to one component or innacurately named